packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -61,14 +61,14 @@ class RenderAndroidView extends RenderBox {
   RenderAndroidView({
     @required AndroidViewController viewController,
     @required this.hitTestBehavior,
-    List<OneSequenceGestureRecognizer> gestureRecognizers = const <OneSequenceGestureRecognizer> [],
+    @required Set<Factory<OneSequenceGestureRecognizer>> gestureRecognizers,
   }) : assert(viewController != null),
        assert(hitTestBehavior != null),
        assert(gestureRecognizers != null),
        _viewController = viewController
   {
     _motionEventsDispatcher = new _MotionEventsDispatcher(globalToLocal, viewController);
-    this.gestureRecognizers = gestureRecognizers;
+    this.gestureRecognizers = gestureRecognizers ?? new Set<Factory<OneSequenceGestureRecognizer>>();
```

Comment:
assuming you never mutate this set, i recommend using a static private so you can always point to the same default empty instance.